retrieveProducts/scripts/dml.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO product(title, description, price, img)
```

Comment:
You do not have a column `img` in `ddl.sql`. This is also a reason why product cannot be created

Reply:
Actually I have. If you take a look at last line of this file you will see that I have added img column
https://github.com/aleksei-bulgak-study/nodejs-aws-be/blob/task4/retrieveProducts/scripts/ddl.sql#L19

Reply:
Yes, then it's fine